### ISSUE TITLE
websocket-client library conflict

### ISSUE THREAD

**Issue by ollo69** [RogerSelwyn/Home_Assistant_SkyQ_MediaPlayer#14]
**Describe the bug**  
This component require websocket-client==0.57.0 but HA use internally websocket-client==0.56.0 for some integration (e.g. Samsung TV).
This is creating library conflict, infact during HA version update this error is showed:

```
pyskyqremote 0.2.23 has requirement websocket-client==0.57.0, but you'll have websocket-client 0.56.0 which is incompatible.
```

Do you really need websocket-client==0.57.0 ?

**Comment by RogerSelwyn** [comment]
I can take a look tomorrow. What version of HA are you using? I don't have this error on 111 or 112 as far as I am aware, or even on 109? That said I'm not sure what log you are looking in to see the error.

**Comment by ollo69** [comment]
I'm using 112.2, but HA is using websocket-client==0.56.0 from a lot.
I now that new version of this library is available, but in HA dependecies was never updated.
The error is only shown during upgrade to new version and only if component that use this library is configured because HA download external library only when required, but in that case the conflict can create issues.

**Comment by RogerSelwyn** [comment]
Ok, I'll try to put less restrictive requirement in tomorrow.

**Comment by RogerSelwyn** [comment]
Fixed by release [v2.2.4](https://github.com/RogerSelwyn/Home_Assistant_SkyQ_MediaPlayer/releases/tag/v2.2.4)
